test/FunctionalTestUtils/TelemetryTestsBase.cs

```diff
@@ -48,5 +52,42 @@ public void ValidateBasicException(InProcessServer server, string requestPath, E
             Assert.NotEmpty(actual.Context.Operation.Name);
             Assert.NotEmpty(actual.Context.Operation.Id);
         }
+
+        public void ValidateBasicDependency(InProcessServer server, string requestPath, DependencyTelemetry expected)
+        {
+
+#if dnx451 
```

Comment:
Let's do this if around entire method so we will not see passed dependency test for dnxcore